types/koa-bodyparser/index.d.ts

```diff
@@ -18,8 +20,8 @@ import * as Koa from "koa";
 
 declare module "koa" {
     interface Request {
-        body: {} | null | undefined;
-        rawBody: {} | null | undefined;
+        body: any;
```

Comment:
Do you mean that body is an object with some set of unknown fields?
In that case, it should be
```ts
body: {
  [key: string]: any
}
```
to be a little stronger.

Reply:
[parseBody](https://github.com/koajs/bodyparser/blob/944870136813136da6bbc86de2167c2762d40a96/index.js#L89-L101) have 4 branches.

* https://github.com/cojs/co-body/blob/db6041c27ce9a6b280aa49c88d82e3ee0da6a844/lib/json.js#L39-L58
* https://github.com/cojs/co-body/blob/db6041c27ce9a6b280aa49c88d82e3ee0da6a844/lib/form.js#L40-L48
* https://github.com/cojs/co-body/blob/db6041c27ce9a6b280aa49c88d82e3ee0da6a844/lib/text.js#L34-L36
* and {}.

So, 

```
type Body = ReturnType<JSON['parse']> | ReturnType<QueryString['parse']> | string | {}
          = any                       | any                              | string | {}
          = any
```

https://github.com/Microsoft/TypeScript/blob/3e4c5c95abd515eb9713b881d27ab3a93cc00461/lib/lib.es5.d.ts#L1049

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c8b8c10fce210a1168422f5589a2442250123255/types/qs/index.d.ts#L47

Maybe we could make it `type Body = {[key: string]: any} | string`,
but I'm not entirely sure. I'd rather follow existing definitions.